dist/pathfora.js

```diff
@@ -2302,6 +2308,28 @@
      */
     this.version = '0.0.04';
 
+    /**
+     * @public
+     * @description callbacks to execute once segments load
+     */
+    this.callbacks = [];
+
+    /**
+     * @public
+     * @description Add callbacks to execute once segments load
+     */
+    this.addCallback = function (cb) {
+      if (context.lio) {
+        if (!context.lio.segmentsCbCalled) {
```

Comment:
instead of adding another new variable to lio we can just use `lio.loaded` it gets set to true after the entity as been loaded. at that point we can just just fire the callback `cb(window.lio.data)`. 

basically if lio has been loaded we dont want to add any more callbacks since there isnt an event to fire them / the data is already accessible.